WORKSPACE

```diff
@@ -15,3 +15,27 @@ envoy_dependencies()
 load("//bazel:dependency_imports.bzl", "envoy_dependency_imports")
 
 envoy_dependency_imports()
+
+load("@rules_python//python:repositories.bzl", "py_repositories")
```

Comment:
@adamkotwasinski this is a quick prototype to demonstrate the use of rules_python. In reality, there is still work needed to fold this into macros like `envoy_dependency_imports`. If we decide to go with this PR (for reasons unrelated to rules_python), I'll split this into a distinct PR.